lib/apple.js

```diff
@@ -356,6 +357,9 @@ function getSubscriptionExpireDate(data) {
     if (!data) {
         return 0;
     }
+    if (data[REC_KEYS.CANCELLATION_DATE_MS]) {
+        return parseInt(data[REC_KEYS.CANCELLATION_DATE_MS], 10);
+    }
```

Comment:
For apple, the purchase is refunded, the expiration date is not changed. But, the cancellation date is logged. So, we should use this date as the expiration date.

https://developer.apple.com/documentation/appstorereceipts/cancellation_date_ms